### DISCUSSION TITLE
Timed Out error for friends

### DISCUSSION THREAD

**Discussion by mardeona** [wolveix/satisfactory-server#239] [Q&A] [answered]
I deployed this this using docker compose and it launched without any errors, managed to claim & name the server without any issues. When I tried to connect, using LAN IP did not work as I was getting the Time Out error, so I used public IP which worked for me perfectly and was able to connect to the server, but my friends can't. They still get the Time Out error, but they can see the server and the ping. All ports (7777, 15000 and 15777) are open. Asked in Satisfactory official discord and was told this is possibly happening due to the server listed with IPv4 and responding with IPv6; Normally, to avoid this issue,  `-multihome` or `-DisablePacketRouting` are used according to the game's Wiki, but I can't see any mention of either in this docker (I also could be blind and missed it somewhere :)).

So is there a way to resolve this issue? Much appreciated

![image](https://github.com/wolveix/satisfactory-server/assets/44360061/d885092b-0ab7-408f-aef2-44a874f92b2d)

**Comment by wolveix** [comment]
Hey @mardeona, please provide your Docker config for the deployment. You can enable `-multihome` by specifying the `SERVERIP` environment variable

**Reply by mardeona** [reply]
Hey @wolveix , I'm not sure which one is Docker config (sorry totally new to most of this stuff). Is it this one:

```dockerfile
version: '3'
services:
    satisfactory-server:
        container_name: 'satisfactory-server'
        hostname: 'satisfactory-server'
        image: 'wolveix/satisfactory-server:latest'
        ports:
            - '7777:7777/udp'
            - '15000:15000/udp'
            - '15777:15777/udp'
        volumes:
            - '/appdata/satisfactoryserver:/config'
        environment:
            - MAXPLAYERS=4
            - PGID=1000
            - PUID=1000
            - ROOTLESS=false
            - STEAMBETA=true
        restart: unless-stopped
        deploy:
          resources:
            limits:
              memory: 6G
            reservations:
              memory: 4G
``` 

And to enable `-multihome` do I assign my public IP to the server?

**Reply by wolveix** [reply, marked as answer]
Hey @mardeona, yep that's what I needed. Add `SERVERIP=0.0.0.0` under your environment variables

**Reply by mardeona** [reply]
That worked! Now my friends are able to connect. Thank you so much for the help!

How about adding in Environment Variables table that `SERVERIP=0.0.0.0` is equivalent to `-multihome`? Assuming it is, I have no knowledge about these things 😄

**Reply by wolveix** [reply]
Glad to hear it! Adding a note within the table isn't a bad idea. Thanks :)
